### PR TITLE
Make nonprofit views translatable

### PR DESCRIPTION
**Pull request by jvBatista**
# Description

Update the `nonprofit` views and make them translatable

### CODE REVIEW

File: app/views/nonprofits/_actions.html.erb [hunk 1]
```diff
@@ -5,18 +5,18 @@
 <aside class='u-marginBottom--15'>
   <% if current_nonprofit_user? %>
 	  <button class='button--jumbo  u-width--full' open-modal='newCampaign'  if-branded='background-color, dark'>
-			Start a Campaign
+			<%= t('nonprofits.actions.start_campaign') %>
     </button>
   <% else %>
     <a class='button u-width--full' if-branded='background-color, dark' target='_blank' href='/peer-to-peer?npo_id=<%=@nonprofit.id %>'>
-      Start a Campaign for <%= @nonprofit.name %>
+      <%= t('nonprofits.actions.start_campaign_for') %> <%= @nonprofit.name %>
```
Comment (wwahammy) [inline app/views/nonprofits/_actions.html.erb:12]: One thing that's important about i18n is making sure you use the right abstraction. For example here, in English, the text is `Start a Campaign For Nonprofit Name`. But in other languages, nonprofit might go first or in a different place in the string. The information on proper abstraction should describe that in https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations.

Also, please change this name to  `nonprofits.page.actions.start_campaign_for` and use `nonprofit_name` as the name of the argument.

File: app/views/nonprofits/_actions.html.erb [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
     </a>
   <% end %>
 	</button>
 
 	<% if current_nonprofit_user? %>
 		<button class='button--jumbo u-width--full' open-modal='newEvent' data-when-confirmed if-branded='background-color, dark'>
-			Create an Event
+			<%= t('nonprofits.actions.create_event') %>
```
Comment (wwahammy) [inline app/views/nonprofits/_actions.html.erb:19]: nonprofits.page.actions.create_event

**Review by wwahammy — CHANGES_REQUESTED** [review]
Thanks so much for your contribution.

So, I really, really like what you're doing  and started reviewing it but I think this is too much to review at once. Especially when you're new to an open source project, it's good to submit small PRs.

At first, please translate a single one of these files at  first. That way we can make sure everything is going smoothly and then you can ramp up from there.

Thanks so much for your contribution, I'm looking forward to working more with you!

**Comment by jvBatista** [comment]
> Thanks so much for your contribution.
> 
> So, I really, really like what you're doing and started reviewing it but I think this is too much to review at once. Especially when you're new to an open source project, it's good to submit small PRs.
> 
> At first, please translate a single one of these files at first. That way we can make sure everything is going smoothly and then you can ramp up from there.
> 
> Thanks so much for your contribution, I'm looking forward to working more with you!

Hello there, first of all I want to thank you for the support and feedback, I really appreciate the review. I'll be sure to follow the recommendations above.

Regarding the suggestion about the size of the PR, should I close this one and open new, smaller ones??

**Comment by wwahammy** [comment]
> Regarding the suggestion about the size of the PR, should I close this one and open new, smaller ones??

In this case, it would make sense to create a new PR with the changes, link back to this PR in the description of the new PR and then close this PR.

**Pull request closed without merging**